src/System.Net.Sockets/src/System/Net/Sockets/SocketAsyncContext.Unix.cs

```diff
@@ -5,28 +5,32 @@
 using Microsoft.Win32.SafeHandles;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.Runtime.CompilerServices;
 using System.Threading;
 
+// Disable unreachable code warning for trace code
+#pragma warning disable CS0162
```

Comment:
You were going to switch to using [Conditional("SOMETHING")] on the trace methods, right?  I'd prefer to not suppress unreachable code warnings in the whole file.